autosynth/git_source.py

```diff
@@ -24,6 +24,25 @@
 from autosynth import git, executor
 
 
+def _strip_pr_number(commit_subject: str) -> str:
```

Comment:
I don't see this being used anywhere except the test

Reply:
Fixed.